external/src/main/java/com/redhat/cloud/custompolicies/engine/TracerStartup.java

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
No copyright headers used in engine. Also, the indentation is broken in this class.